### ISSUE TITLE
master signal

### ISSUE THREAD

**Issue by 3nids** [opengisch/signalo#92]
filter master signal by azimuth

https://app.clickup.com/t/2192114/DEV-5285

**Comment by 3nids** [comment]
facing a QGIS issue: it might be a bug with self-referecing layer + filter. Trying to create a minimal reproducible example.

**Comment by itsakifa** [comment]
self-referencing seems not a problem finally - use the expression : `fk_frame =  current_value( 'fk_frame')` for the filter

**Comment by itsakifa** [comment]
it's probably best to keep "id" as 'Colonne de valeurs'.
![image](https://github.com/opengisch/signalo/assets/4679915/dbda3834-c94b-4213-8337-a0d7043caf62)
 It's not possible to put an expression there. And choosing another attribut would mean needing to choose between the "official_signs" and the "user_signs".

**Comment by itsakifa** [comment]
https://github.com/opengisch/signalo/pull/118/commits/1bf69f49f41bb32f844efbb10523f12ff23d9b5d

**Comment by spch-GL** [comment]
Cela a disparu de la version 1.0.0RC5, est-ce normal ?

**Comment by itsakifa** [comment]
@spch-GL - cela devrait marcher avec la RC6. C'est correct?

**Comment by spch-GL** [comment]
Non, pendant la saisie la liste est vide, et après enregistrement, à l'édition:
![image](https://github.com/opengisch/signalo/assets/26377323/0cf1a820-8ab9-4791-b5aa-e5187217f90b)
il pointe sur lui-même

**Comment by itsakifa** [comment]
@spch-GL ce n'est pas le comportement voulu? D'après ce que nous avons compris, cette liste permet de choisir le panneau principal du cadre. S'il n'y a qu'un signal dans le cadre, il n'y aura que celui là dans la liste.

**Comment by spch-GL** [comment]
Pour le SPCH, il faudrait effectivement filtrer sur l'azimut, car il est courant d'avoir la plaque complémentaire dans un cadre indépendant du signal principal
